product/models.py

```diff
@@ -0,0 +1,91 @@
+from django.db import models
+from order.models import *
+from board.models import *
+from user.models import *
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+    class Meta:
+        db_table = "categories"
+
+class Subcategory(models.Model):
+    name     = models.CharField(max_length=45)
+    category = models.OneToOneField(
+                'Category',
+                on_delete   = models.CASCADE,
+                )
+    class Meta:
+        db_table = "subcategories"
+
+class Character(models.Model):
+    name        = models.CharField(max_length=45)
+    image_url   = models.CharField(max_length=2000, default='')
+    class Meta:
+        db_table = "characters"
+
+class Product(models.Model):
+    name            = models.CharField(max_length=45, null=True)
+    price           = models.DecimalField(max_digits= 6, decimal_places=2)
+    star_rating     = models.DecimalField(max_digits= 1, decimal_places=1)
+    description     = models.TextField()
+    category        = models.OneToOneField(
+                        'Category',
+                        on_delete   = models.CASCADE,
+                        )
```

Comment:
```suggestion
    category        = models.ForeignKey('Category', on_delete   = models.SET_NULL, null=True)
```
위경우에는 사실상 하나를 참조하는것은 맞지만 다수중 하나를 선택하는것이기 때문에 ForeignKey로 하는 것이 맞습니다. OneToOne은 대상 테이블과 현재 테이블 간에  단일요소간의 연결입니다. 개념이 조금 달라요! 그래서 다수의 데이터중 단순연결은 원투원보다는 ForeignKey사용을 해주세요. 이부분도 판단후에 알맞게 모든 필드에 적용 부탁드립니다.

Reply:
@ulfrid